README.rdoc

```diff
@@ -78,7 +77,7 @@ There's also {a Google+ group}[http://goo.gl/TFy1U] and a
 want to chat about something.
 
 If you want to install as a signed gem, the public certs of the gem signers
-can be found in {the `certs` directory}[https://bitbucket.org/ged/ruby-pg/src/tip/certs/]
+can be found in {the `certs` directory}[https://github.com/ged/ruby-pg/tree/master/certs]
```

Comment:
It might be possible to use a [relative link](https://help.github.com/en/articles/about-readmes#relative-links-and-image-paths-in-readme-files) here, `[certs]` or `[certs/]`.

Reply:
Relative links would only work when rendered on Github, right?

Reply:
Yes, true. The word "directory" made me think of a local checkout.

Since this is about installation, an absolute URL makes sense.